src/spaceObjects/spaceship.h

```diff
@@ -312,6 +313,8 @@ class SpaceShip : public ShipTemplateBasedObject
     void setEnergy(float amount) { if ( (amount > 0.0) && (amount <= max_energy_level)) { energy_level = amount; } }
     float getSystemHealth(ESystem system) { if (system >= SYS_COUNT) return 0.0; if (system <= SYS_None) return 0.0; return systems[system].health; }
     void setSystemHealth(ESystem system, float health) { if (system >= SYS_COUNT) return; if (system <= SYS_None) return; systems[system].health = std::min(1.0f, std::max(-1.0f, health)); }
+    float getSystemHealthMax(ESystem system) { if (system >= SYS_COUNT) return 0.0; if (system <= SYS_None) return 0.0; return systems[system].health_max; }
+    void setSystemHealthMax(ESystem system, float health_max) { if (system >= SYS_COUNT) return; if (system <= SYS_None) return; systems[system].health_max = std::min(1.0f, std::max(0.0f, health_max)); }
```

Comment:
As health is from -1.0 to 1.0, shouldn't max health be allowed negative as well?

Reply:
I thought about 0.0 when the system is completely broken, but yes, a consistency with health is better. So I need to understand how to accept a review into a commit ^^ (and that change the tweak too)

Reply:
After some tests, it is hard to visualize negative health max with the black bar. Maybe I can replace black bar with an icon when the maximal health is below 1.0. What do you think ?

![image](https://user-images.githubusercontent.com/13868149/81936208-2b577e80-95f2-11ea-8b5c-c3709e02a729.png)